lighthouse-core/gather/driver.js

```diff
@@ -433,12 +433,14 @@ class Driver {
    *    See https://developer.mozilla.org/en-US/docs/Web/API/Permissions/query.
    */
   queryPermissionState(name) {
+    if (['geolocation', 'notifications', 'midi', 'push'].indexOf(name) === -1) {
```

Comment:
The API already rejects for cases it doesn't understand:

![screen shot 2016-11-28 at 5 13 45 pm](https://cloud.githubusercontent.com/assets/238208/20692629/16962d86-b58e-11e6-8fce-d9a445812d74.png)